src/openrct2/management/Marketing.cpp

```diff
@@ -88,7 +88,7 @@ static void marketing_raise_finished_notification(const MarketingCampaign& campa
             ft.Add<StringId>(GetShopItemDescriptor(campaign.ShopItemType).Naming.Plural);
         }
 
-        News::AddItemToQueue(News::ItemType::Money, MarketingCampaignNames[campaign.Type][2], 0, ft);
+        News::AddItemToQueue(News::ItemType::Campaign, MarketingCampaignNames[campaign.Type][2], campaign.Type, ft);
```

Comment:
Is it still necessary to pass `campaign.Type` here, now that you changed your PR to just link to the overview page?

Reply:
It's not. It can't do any harm, but let's prevent any confusion about whether something is actually done with it (which there isn't).

Reply:
It seems there are not any other built-in notfications that use the money item type which surprises me. But plugins may do and expect it to link to the main finance page. This also allows you to have a different icon for marketing.

Reply:
If a plugin uses the money type, the main finance page will still be opened. I don't see an issue there!

Yes, we could use a separate icon (right now both Money and Campaign use the same), but I didn't find anything better that specifies campaigns even more.